pkg/placement/placement_test.go

```diff
@@ -30,6 +31,18 @@ func init() {
 
 func TestGetAddresses(t *testing.T) {
 	testAddress := "127.0.0.1"
+	ipAddressType := v1beta1.IPAddressType
+	addresses := []v1beta1.GatewayAddress{
```

Comment:
one question here, is it worth testing with more than 1 IP? The main issue we had was around multiple addresses.

Reply:
Yes.
I'll push an update.

Reply:
Pushed.
I intend to do additional testing that hits this in the form on integration tests as part of implementing #275